src/components/transport_manager/include/transport_manager/transport_adapter/transport_adapter_impl.h

```diff
@@ -91,7 +91,7 @@ class TransportAdapterImpl : public TransportAdapter,
    * @return true if initialized.
    * @return false if not initialized.
    */
-  virtual bool IsInitialised() const;
+  virtual bool IsInitialised() const OVERRIDE;
```

Comment:
I did this as requested by @AByzhynar - see comment above. IsInitialised() is pure virtual method from TransportAdapter, implemented in TransportAdapterImpl. So I supposed that in this case the use of OVERRIDE is appropriate. Please correct me if I am wrong.